networks/remote/ansible/roles/config/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+- name: Copy binary
+  copy:
+    src: "{{BINARY}}"
+    dest: /usr/bin
+    mode: 0755
+
+- name: Copy config
+  when: item <= N and ansible_hostname == 'remotenet-node' ~ item
+  copy:
+    src: "{{CONFIGDIR}}/node{{item}}/"
+    dest: "/home/{{service}}/.{{service}}/"
+    owner: "{{service}}"
+    group: "{{service}}"
+  loop: [ 0, 1, 2, 3, 4, 5, 6, 7 ]
```

Comment:
why 8 if you're creating only 4 machines?

Reply:
The ansible role can take up to 8 nodes (as an upper limit). This means that when you start the script, it uses N=4 as a default and the rest will not be used, but if you want to run 5 nodes, you can still do it. Unfortunately there's no easy way in Ansible to have an unbound upper limit for these kind of loops.

Reply:
strange, but ok.